pom.xml

```diff
@@ -133,25 +119,38 @@
             <url>https://maven.forgerock.org:443/repo/community</url>
         </repository>
         <repository>
-            <id>jcenter</id>
-            <url>https://jcenter.bintray.com/</url>
-        </repository>
-        <repository>
+            <id>central</id>
```

Comment:
Why here and not in the forgerock artifactory at maven.forgerock.org?

Reply:
The central maven repository it's for third parties the forgerock maven repository it's for forgerock packages, we don't use forgerock proxy maven repository.

Reply:
I have deleted the maven.forgerock repositories because this repo doesn't get any forgerock dependency.

Reply:
Oh yes, of course! Sorry!